app/views/users/_post.html.erb

```diff
@@ -4,9 +4,13 @@
     <%= post.text%><br>
     <div class='likes-comments-counters'>
         <%= form_with url: user_post_likes_path(post_id: post.id, user_id: user.id ), method: :post do |f| %>
-        <%=f.hidden_field :post_id, value: post.id %>
+        <%= f.hidden_field :post_id, value: post.id %>
         <%= f.submit :Like %> 
-        <%end%>
+        <%end%><br/>
+            <%= form_with url: user_post_path(user, post), method: :delete do |f| %>
+            <%= f.hidden_field :post_id, value: post.id %>
+            <%= f.submit :Delete %> 
+            <%end%><br/>
```

Comment:
- [ ] Good job on applying the authorization properly! 💯   However, as you can read below, I shouldn't see the delete buttons if I am not authorized to do. Kindly add this functionality as well. 
![image](https://user-images.githubusercontent.com/91486791/218175814-9aa82f46-ea72-4791-85e2-6456b42210c0.png)
![image](https://user-images.githubusercontent.com/91486791/218175858-b7b05aee-ab81-4afb-a000-2e360b8265a0.png)
![image](https://user-images.githubusercontent.com/91486791/218176020-c8768175-b0ad-4229-b090-e29149a02041.png)

Reply:
I did it, Thank you so much